services/gitea/9.3.0/defaults/cm.yaml

```diff
@@ -84,7 +84,7 @@ data:
       primary:
         priorityClassName: "dkp-critical-priority"
       image:
-        tag: 11.16.0-debian-11-r9
+        tag: 15.2.0-debian-11-r21
```

Comment:
What other entity is downloading this specific image? Also, can we just remove the override and use whatever is the upstream default (which seems to be `15.4.0-debian-11-r10` looking at the chart url)

Reply:
It's being installed by dkp-insights. I'll remove the override and see if I can update dkp-insights to use the same.

Reply:
since we downgraded gitea, I decided to just revert the postgres image overrides. we'll just let insights use their default image in case this downgraded postgres image isn't compatible